WooCommerce/Classes/ViewRelated/Dashboard/Settings/Domains/DomainSelectorDataProvider.swift

```diff
@@ -52,12 +52,37 @@ struct PaidDomainSuggestionViewModel: DomainSuggestionViewProperties, Equatable
 
     init(domainSuggestion: PaidDomainSuggestion) {
         self.name = domainSuggestion.name
-        // TODO: 8558 - attributed price info
-        self.attributedDetail = .init("\(domainSuggestion.saleCost ?? "no sale") / \(domainSuggestion.cost) / \(domainSuggestion.term)")
+        self.attributedDetail = {
+            var attributedCost = AttributedString(.init(format: Localization.priceFormat, domainSuggestion.cost, domainSuggestion.term))
+            attributedCost.font = .body
+            attributedCost.foregroundColor = .init(.secondaryLabel)
+
+            if let saleCost = domainSuggestion.saleCost {
+                attributedCost.strikethroughStyle = .single
```

Comment:
❓ Should we strike through the whole string including the term? i.e. year
I am wondering whether we should only strikethrough the cost and keep the term unedited.
![Uploading StrikeThrough.jpg…]()

Reply:
Great catch! I followed the design that crosses out the whole string including the term, but I agree that it's more clear to exclude the term to avoid any misunderstanding by the merchants. Updated the screenshots in the PR description with changes in https://github.com/woocommerce/woocommerce-ios/pull/8710/commits/b31dfbfab5050753b5ff74c64d27bc6931d1d1b4